ansible/README.md

```diff
@@ -1,60 +1,13 @@
-OMERO VM Deployment
-===================
+OME Ansible 
+===========
 
-This contains example files for provisioning an OpenStack VM from scratch with OMERO using Ansible.
+This contains a variety of Ansible playbooks and roles, including example files for provisioning an OpenStack VM from scratch with OMERO using Ansible.
 Most of these scripts should also work on other platforms, providing the VM is brought up by some other method.
-The guest must be running CentOS 7.
 
+- Playbooks which start with `os` are OpenStack specific - the `os` stands for `openstack`.
+- Playbooks which start with `idr` are for the Image Data Repository.
+- Playbooks which start with `ci` are for OME continuous integration and build.
 
-Guide for the Impatient
------------------------
+HERE LINK TO: [[README-os.md]]
 
-Setup your OpenStack environment variables, and run:
-
-    ansible-playbook os-create.yml -e omero_vm_name=FOO -e omero_vm_key_name=YOURKEY
-    ansible-playbook -i inventory -l 'FOO' os-omero.yml
-
-Some of the more specific deployment scripts may work depending on your environment:
-
-    ansible-playbook -i inventory -e omero_vm_name=FOO -e omero_vm_key_name=YOUR_KEY uod-os-idr-omero.yml
-
-
-`os-omero.yml`
---------------
-
-This is the Ansible playbook that will be run to setup OMERO.
-You can also run it manually to install OMERO on localhost.
-
-
-`os-create.yml`
----------------
-
-This playbook will connect to OpenStack and spin up a VM.
-The Ansible modules in this playbook require the `shade` Python module.
-
-Before running the playbook you must [setup your OpenStack environment variables](http://docs.openstack.org/user-guide/common/cli_set_environment_variables_using_openstack_rc.html).
-You can override variables at the command line, for example (note double quoting is necessary if spaces are present):
-
-    ansible-playbook os-create.yml -e omero_vm_name=FOO -e omero_vm_key_name=YOURKEY \
-        -e "omero_vm_flavour='m2.xxlarge'"
-
-If this step fails it could be due to an incorrect variable, the Ansible `os_server` module usually gives an uninformative "Error in creating instance" message.
-If the VM was created the floating IP of the VM will be printed out.
-
-To delete the VM and related security group:
-
-    ansible-playbook os-delete.yml -e omero_vm_name=FOO
-
-If another instance is using the OMERO security group, the task will fail but can be safely ignored.
-
-Inventory
----------
-
-This directory takes advantage of an
-[Ansible dynamic inventory script for OpenStack (`openstack.py`)](http://docs.ansible.com/ansible/intro_dynamic_inventory.html#example-openstack-external-inventory-script)
-instead of having to manage an inventory file when using Ansible to push out changes.
-For example:
-
-    ansible-playbook -i inventory -l os-image-centos os-omero.yml -vv
-
-Variables for the groups defined in os-create.yml as omero_vm_groups can be added under inventory/variables.
+HERE LINK TO:  
```

Comment:
Yeah, sorry: was supposed to actually link to the files I'd effectively replaced by this general README. Updated now. Was in the "have to get a PR with the playbook in" to be able to comment on a PR with the output of the plays.. circular holding pattern..